### PR TITLE
Accessbility project

### PR DESCRIPTION
**Pull request by SabaAhmad404**
I did this project with my coding partner and did some accessibility, and add the following:
 - Page titles - no issues found
 - Image text alternatives - no issues found
 - Text heading - no issues found
 - Color Contrast, [changed the color of form button](https://github.com/Abdessittir/Abdessittir.github.io/blob/accessbility/style.css#L334)
 - Resize -no issues found
 - Interaction - no issues found
 - Moving Content - no issues found
 - Multi-media - no issues found
 - The basic structure of the page - no issues found

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -326,6 +326,13 @@
     height: 48px;
     color: #fff;
     cursor: pointer;
```
Comment (andgarzonmal) [inline style.css:328]: - [ ] Hello team you did an excellent job on the project 📈,but your report should list the result of the evaluation you did. PLease remember to add it on the description of your pull request. ♻️

image

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @SabaAhmad786✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️


### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css (hunk 1 above)
Comment (AHP15) [reply style.css:328]: Hi @andgarzonmal thanks for the feedback

**Review by andgarzonmal — APPROVED** [review]
Hi @SabaAhmad786, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**